src/App.scss

```diff
@@ -20,3 +20,22 @@
     width: 1.2rem;
   }
 }
+
+
+.icon-button-disabled {
+  background-color: transparent !important;
+  border-color: transparent !important;
+
+  &:focus {
+    background-color: transparent !important;
+  }
+  
+  &.checked {
+    background-color: transparent !important;
+    border-color: transparent !important;
+  }
+
+  & > img {
+    width: 1.2rem;
+  }
+}
```

Comment:
Good to simplify this as just an image, not a button.  If it is just a button, can probably get away with just the `width` property and remove the rest?  See if we can simplify this.

Also, can you fix my mistake and move these 2 SCSS rules to the big CSS file?  We will just use SCSS to compile the pyData theme.

Reply:
Yes, I am on it.